Project/src/MakeCall/MakeCall.js

```diff
@@ -230,6 +231,21 @@ export default class MakeCall extends React.Component {
         }
     };
 
+    joinRooms = async (withVideo) => {
+        try {
+            if(this.roomsId.value) {
+                const callOptions = await this.getCallOptions(withVideo);
+                this.callAgent.join({ meetingId: this.roomsId.value }, callOptions);
+            } else {
+                throw new Error('Please enter Rooms ID to join a Rooms call');
+            }
+

```

Comment:
you need to render that error not throw it

Reply:
Fixed.